index.html

```diff
@@ -438,6 +438,41 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model properties and types in this specification map to URLs.
+The vocabulary where these URLs are defined is the [[?SECURITY-VOCABULARY]]. The
+explicit mechanism that is used to perform this mapping in a secured document is
+the `@context` property.
+      </p>
+      <p>
+When processing a document in a JSON-LD environment, the mapping mechanism is
+defined by JSON-LD [[JSON-LD11]]. When processing a document in a non-JSON-LD
+environment, the same mechanism is used to ensure that the terms have the same
+interpretation regardless of format, but with a simpler processing rule:
```

Comment:
```suggestion
interpretation regardless of format, but with a simpler processing rule:
```

(Editorial) I am not sure why you end the sentence with the ':' character. I _presume_ you meant the two paragraph following this one to be in a bullet list (in which case the `<ul>...` markup is missing).

Reply:
No, it really is meant to only be the next paragraph. That said, I see why the way it is written now would be confusing. I'll update the section in an attempt to remove the confusing way it is set up right now.

Reply:
Fixed in 02deef295eb69fcca67c38978756a87539299d30.

Reply:
Further revisions to the language in 0582d45.